internal/ethereum/ethereum_service.go

```diff
@@ -197,9 +197,9 @@ func (service ethereumService) Update(dto *EthereumDto, node *ethereumv1alpha1.N
 	}
 	if dto.Engine != nil {
 		node.Spec.Engine = *dto.Engine
-	}
-	if dto.JWTSecretName != "" {
-		node.Spec.JWTSecretName = dto.JWTSecretName
+		if dto.JWTSecretName != "" {
+			node.Spec.JWTSecretName = dto.JWTSecretName
+		}
```

Comment:
this is the same result ! please update the code.
`dto.engine` can be not nil, but false.

check this fragment.
```go
        if dto.GraphQL != nil {
		node.Spec.GraphQL = *dto.GraphQL
	}
	if node.Spec.GraphQL {
		if dto.GraphQLPort != 0 {
			node.Spec.GraphQLPort = dto.GraphQLPort
		}
	}
```